operator/scheduler/client.go

```diff
@@ -34,6 +35,9 @@ const (
 	// these 2 constants in combination with the backoff exponential function will give us a max backoff of 13.5 minutes
 	SchedulerConnectMaxRetries    = 12
 	SchedulerConnectBackoffScalar = 200 * time.Millisecond
+	ClientKeapAliveTime           = 10 * time.Second
```

Comment:
The [docs](https://grpc.io/docs/guides/keepalive/) say: "To avoid DDoSing, it’s important to take caution when setting the keepalive configurations. Thus, it is recommended to avoid enabling keepalive without calls and for clients to avoid configuring their keepalive much below one minute."

I suppose we'd not be ddos-ing ourselves with 10s and just a couple of services -- although it would depend on the number of agents connecting, but we'd probably need to test this value thouroughly before the next release.